open.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+sem=$1
+str_num=$2
+
+#positive=("Y" "y" "Yes" "yes")
+
+echo -n "Открыть электронный корявник? (Д/н): "
+read ans
+    
+if [[ $ans == "y"* || $ans == "Y"* || $ans == "д"* || $ans == "Д"* ]]; then
+
+    if [ ! -f "KORYAVNIKS/${sem}.djvu" ]; then
+    
+        mkdir -p KORYAVNIKS/
+        ./download.sh $sem
+      
+    fi
+     
+     #here you can place your pdf viewer script
+    ./okular.sh $sem $str_num
```

Comment:
If some additional software is required for the script to work properly, a dependencies must be specified to user in README. Also, you should check if the certain dependency is met, for instance, in this particular example one would do "okular --version" or something like this.

Reply:
I positioned this script as an example and it actually works out-of-the-box with standart KDE software pack (cause Okular seems to be a part of it). I suppose that there will be other users who implement support for other document viewers.
Version checking seems to be a good idea for installation script or for end user's document opening script. I'll definitely add it to `okular.sh` or to installation script depending on the final `open.sh` concept.